File: mathematics/multiplication-tables.cpp
#include <iostream>

using namespace std;

int main()
{
  int number;

  // Get input from the user
  cout << "Enter the number to print table: ";
  cin >> number;

  // Print the multiplication table
  for (int i = 1; i <= 10; ++i)
  {
    cout << i << " x " << number << " = " << (number * i) << endl;
  }

  return 0;
}

File: mathematics/multiplication-tables.py
num = int(input("Enter the number to print table: "))
i = 1

while i <= 10:
    print(i, "x", num, "=", num * i)
    i += 1

"""
# multiplication table using for
number = int(input("Enter the number: "))

print(f"\nMultiplication Table for {number}:\n")
for i in range(1, 11):
    print(f"{i} x {number}  = {number * i}")
    i += 1
"""